### PR TITLE
#2 added galaxy metadata still missing deps

### PR DESCRIPTION
**Pull request by Scriptkiddi**

### CODE REVIEW

File: meta/main.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 galaxy_info:
-  author: your name
-  description: your description
-  company: your company (optional)
+  author: Fritz Otlinghaus
+  description: Ansible Playbook for Icinga2 
+  company: stuvus-IT
```
Comment (SlothOfAnarchy) [inline meta/main.yml]: use our ansible sceleton (e.g. the link is missing)

File: meta/main.yml [hunk 2]
```diff
@@ -32,10 +32,10 @@ galaxy_info:
...
   # platforms is a list of platforms, and each platform has a name and a list of versions.
   #
-  # platforms:
-  # - name: Fedora
-  #   versions:
-  #   - all
+  platforms:
+   - name: Ubuntu-Server
+     versions:
+     - 16.04
```
Comment (SlothOfAnarchy) [inline meta/main.yml]: use version name

File: meta/main.yml [hunk 3]
```diff
@@ -44,7 +44,7 @@ galaxy_info:
   #   - 7
   #   - 99.99
 
-  galaxy_tags: []
+  galaxy_tags: [icinga2, icinga, stuvus]
```
Comment (SlothOfAnarchy) [inline meta/main.yml]: please add monitoring tag

File: meta/main.yml [hunk 4]
```diff
@@ -1,7 +1,7 @@
 galaxy_info:
-  author: your name
-  description: your description
-  company: your company (optional)
+  author: Fritz Otlinghaus
+  description: Ansible Playbook for Icinga2
```
Comment (SlothOfAnarchy) [inline meta/main.yml]: I think you meant role

File: meta/main.yml [hunk 5]
```diff
@@ -1,57 +1,26 @@
...
-  # - MIT
-  # - GPLv2
-  # - GPLv3
-  # - Apache
-  # - CC-BY
-  license: license (GPLv2, CC-BY, etc)
+  min_ansible_version: 2.2
 
-  min_ansible_version: 1.2
+  github_branch: master
```
Comment (dasJ) [inline meta/main.yml]: Brauchst du nicht

Comment (Scriptkiddi) [inline meta/main.yml]: done

**Pull request merged**